rfcs/52-unstable_features.md

```diff
@@ -0,0 +1,89 @@
+# Feature Name: `unstable_features`
+
+## Summary
+
+Experimental new features are merged into main behind a feature gate before stabilization.
```

Comment:
I would propose we call these experimental features something other than "unstable", since a "stable feature" implies that we won't change the API of that feature going forwards; we don't make that guarantee about basically anything in Bevy at this point.

Rust has this to say about their own "unstable/stable" feature process:
> Code that works and runs on stable should not break.

> Until [it has been stabilized], the feature is not set in stone: every part of the feature can be changed, or the feature might be completely rewritten or removed.

Even a "stable feature" under the text of this RFC, as with almost all features in Bevy right now, is subject to change and might be completely rewritten or removed.

Plenty of good alternatives: `experimental`, `prototype`, `canary`, `proto`, `future`, et c.

Reply:
prototype is my favorite of those. Definitely like it better than unstable in the context of Bevy.

Reply:
I like it too - especially given the parallel to `bevy_prototype_<name>` community crates which, from other discussion here, it sounds like will still be encouraged where possible.

Reply:
This is a valid point and I’d like to cast my vote on `experimental`. I think the others are a bit more vague/formal/rigid whereas `experimental` feels more approachable. To me it says, "this is an _experiment_ 🧪 to gain feedback and toy around with implementations to see if we (as a community) want to pursue this feature or a feature like this."